src/raw.rs

```diff
@@ -292,6 +303,47 @@ pub fn get_string() -> io::Result<String> {
     }
 }
 
+/// Retrieves a list of file paths from the `CF_HDROP` format.
+///
+/// # Pre-conditions:
+///
+/// * [open()](fn.open.html) has been called.
+pub fn get_file_list() -> io::Result<Vec<PathBuf>> {

```

Comment:
Is it possible to get us iterator instead of `Vec`?

Reply:
I don't think that's much better since we'd have to copy the clipboard data to return an iterator. We already have to allocate all the PathBufs, so whether we have to copy the clipboard data or allocate an extra `Vec` doesn't seem like much of a difference.

Reply:
ok